File: app/src/main/java/com/example/brochures/network/parsingobjects/Publisher.java
package com.example.brochures.network.parsingobjects;

import com.squareup.moshi.Json;

public class Publisher {

    @Json(name = "id")
    private Integer id;
    @Json(name = "name")
    private String name;
    @Json(name = "type")
    private String type;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/main/java/com/example/brochures/network/parsingobjects/Publisher.kt
package com.example.brochures.network.parsingobjects

import com.squareup.moshi.Json

class Publisher {
    @Json(name = "id")
    var id: Int? = null

    @Json(name = "name")
    var name: String? = null

    @Json(name = "type")
    var type: String? = null
}